index.mjs

```diff
@@ -77,4 +77,35 @@ export const buffer = stream => Future ((rej, res) => {
   return removeListeners;
 });
 
+//# instant :: b -> Future a b
+//.
+//. Resolves a Future with the given value in the next tick,
+//. using `process.nextTick`. The scheduled job cannot be cancelled
+//. and will run before any other jobs, effectively blocking the
+//. event loop until it's completed.
+//.
+//. ```js
+//. > instant ('noodles')
+//. Future.of ('noodles')
+//. ```
+export const instant = x => Future ((rej, res) => {
+  process.nextTick (res, x);
+});
+
+//# immediate :: b -> Future a b
+//.
+//. Resolves a Future with the given value in the next tick,
+//. using `setImmediate`. This job will run as soon as all
+//. other jobs are completed. Cancelling the Future un-schedules
+//. the job.
+//.
+//. ```js
+//. > immediate ('results')
+//. Future.of ('results')
+//. ```
+export const immediate = x => Future ((rej, res) => {
+  const immediate = setImmediate (res, x);
```

Comment:
```suggestion
  const id = setImmediate (res, x);
```

Reply:
I didn't use `id` because technically it returns more than an `id`. Using the same name as the function is confusing though. I changed it to `job` - what do you think?

Reply:
:thumbsup: